### PR TITLE
Sort jars list before copying to fix inconsistent jar reporting

### PR DESCRIPTION
**Pull request by pramodya1994**
- Fixes https://github.com/ballerina-platform/ballerina-lang/issues/34942

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BuildCommandTest.java [hunk 1]
```diff
@@ -656,39 +656,18 @@ public void testBuildWithInvalidOrgName() throws IOException {
...
 
-            Files.copy(file, toPath.resolve(fromPath.relativize(file).toString()), copyOption);
-            return FileVisitResult.CONTINUE;
+        // Check build output which contains conflicted jars for 10 consecutive builds
+        for (int i = 0; i < 10; i++) {
+            BuildCommand buildCommand = new BuildCommand(projectPath, printStream, printStream, false, false);
+            new CommandLine(buildCommand).parse();
+            buildCommand.execute();
+            String buildLog = readOutput(true);
+            Assert.assertEquals(buildLog.replaceAll("\r", ""), getOutput("build-conflicted-jars-project.txt"));
```
Comment (azinneera) [inline cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BuildCommandTest.java:670]: I think build command should be executed multiple times to ensure the consistency of the list

Comment (pramodya1994) [reply cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BuildCommandTest.java:670]: We execute the build command for 10 consecutive builds.

**Review by azinneera — APPROVED** [review]

**Comment by pramodya1994** [comment]
Build errors will get fixed after [this PR](https://github.com/ballerina-platform/ballerina-lang/pull/35361).

**Comment by pramodya1994** [comment]
Blocked in https://github.com/ballerina-platform/ballerina-lang/issues/35608

**Comment by pramodya1994** [comment]
Added to `master` branch via https://github.com/ballerina-platform/ballerina-lang/pull/35784

**Pull request closed without merging**